.github/workflows/my-ci-pipeline.yml

```diff
@@ -36,15 +36,6 @@ jobs:
           target: production
           push: true
           tags: ${{ secrets.DOCKER_USERNAME }}/todo_app:latest, ${{ secrets.DOCKER_USERNAME }}/todo_app:${{ github.sha }}
-      - name: Login to Heroku
-        env:
-          HEROKU_API_KEY: ${{ secrets.HEROKU_API_KEY }}
-        run: heroku container:login
-      - name: Push to Heroku
-        run: |
-          docker build --target production --tag registry.heroku.com/todo-app-by-wendi/web .
-          docker push registry.heroku.com/todo-app-by-wendi/web
-      - name: Release
-        env:
-          HEROKU_API_KEY: ${{ secrets.HEROKU_API_KEY }}
-        run: heroku container:release web -a todo-app-by-wendi
\ No newline at end of file
+      - name: Deploy to Azure
+        run: curl -dH -X POST ${{secrets.AZURE_WEBHOOK_URL}}
```

Comment:
One nice addition here can be to add the [--fail-with-body](https://daniel.haxx.se/blog/2021/02/11/curl-fail-with-body/#:~:text=%2D%2Dfail%2Dwith%2Dbody%20will,code%20was%20400%20or%20larger.&text=If%20the%20page%20is%20missing,Not%20complicated%20at%20all.) argument, because currently if Azure were to return an error response, curl would just assume getting any response was a success which can lead to some confusing pipeline behaviour!

Reply:
This command seems to be not found - https://github.com/WendiF/DevOps-Course-Starter/runs/7861440321?check_suite_focus=true

Is this perhaps because it's a version of curl that doesn't support the command? I'm not sure what the best way to diagnose/debug this problem is

Reply:
Hmm, happy with your call to skip it but for closure my thoughts are:
* Your pipeline output makes it look slightly like it's interpreted it as a separate command, so potentially moving it before the webhook URL would work?
* If it is a newer/different version of curl, then there's a `--fail` argument that is slightly older that should work (it works a little differently, but achieves a very similar thing)

> Run curl -dH -X POST ***
>  curl -dH -X POST ***
 > --fail-with-body

Reply:
I've also tried adding it as 'run: curl --fail-with-body -dH ....' but got a similar error - was wondering if there's any way to know what version of curl docker is running (other than perhaps adding a specific line to print that??) Can give the --fail flag a go :)